firstvoices/backend/serializers/job_serializers.py

```diff
@@ -36,28 +31,23 @@ class Meta:
         )
 
 
-class CustomOrderRecalculationResultSerializer(serializers.ModelSerializer):
-    site = serializers.StringRelatedField()
-    current_task_status = serializers.SerializerMethodField()
-    latest_recalculation_result = serializers.SerializerMethodField()
-
-    @staticmethod
-    @extend_schema_field(OpenApiTypes.STR)
-    def get_current_task_status(obj):
-        async_result = AsyncResult(obj.task_id)
-        return async_result.status
+class DictionaryCleanupJobSerializer(
+    CreateSiteContentSerializerMixin, BaseJobSerializer
+):
+    url = SiteHyperlinkedIdentityField(
+        read_only=True, view_name="api:dictionary-cleanup-detail"
+    )
+    is_preview = serializers.BooleanField(read_only=True)
+    cleanup_result = serializers.SerializerMethodField(read_only=True)
 
     @staticmethod
     @extend_schema_field(OpenApiTypes.STR)
-    def get_latest_recalculation_result(obj):
-        if (
-            hasattr(obj, "latest_recalculation_result")
-            and obj.latest_recalculation_result
-        ):
-            unknown_character_count = obj.latest_recalculation_result.get(
+    def get_cleanup_result(obj):
+        if hasattr(obj, "cleanup_result") and obj.cleanup_result:
+            unknown_character_count = obj.cleanup_result.get(
                 "unknown_character_count", 0
             )
-            updated_entries = obj.latest_recalculation_result["updated_entries"]
+            updated_entries = obj.cleanup_result["updated_entries"]
```

Comment:
Would it make sense to put separate fields for updated_entries and unknown_character_count etc directly on the model? Might make the API responses easier to read

Reply:
I realize this is a mischief suggestion-- if other folks think it is a good idea we should log a separate ticket

Reply:
Rather than logging a separate ticket, will add to dictionary cleanup memory changes.

Reply:
Details added to this JIRA: https://firstvoices.atlassian.net/browse/FW-5933